### PR TITLE
Closes #1858 follow up atlases ro is

### LINKED ISSUE #1858: Follow up of issue 900 for fixing atlases & supratentorial
### Description

This issue solves some follow-up work after #900. Some of this was planned in #1740 which was outdated.

### Tasks

- [x] Create Atlases4ROIs/LicensePermissive/SPM_CAT_GPL/README.md from old ROI location
- [x] Remove TotalWM, this is the same as WholeBrain but was used for WM, this is now obsolete with x.S.TissueMasking

We use TotalGM.nii:
1. Comparison/ReferenceValues
2. xASL_qc_SortBySpatialCoV
3. All users are used to having TotalGM output; renaming this to WholeBrain_GM will confuse them

We don't really use WholeBrain.nii. We can keep it for now, and remove it later.

Easy suggestion:
Renaming TotalGM.nii -> Total.nii
Output name should then instead of Total_GM be TotalGM, and the rest stays the same

- [x] Upon loading atlases and finding TotalGM or TotalWM, replace this by x.S.Atlas = 'Total' & x.S.TissueMasking = 'GM' or 'WM'
HENK: so I now fixed that it always adds the TissueMaskingLocal{1} to the ROI name.
E.g.,
ROIname = Total
TissueMasking = GM
-> TotalGM
ROIname = DeepWM
TissueMasking = WM
-> DeepWMWM
ROIname = Supratentorial
TissueMasking = WM
-> SupratentorialWM
ROIname = Hammers
TissueMasking = GM
-> HammersGM

so only for DeepWM this now is DeepWMWM so needs some changes in our testing code etc, but otherwise this seems most optimal

### Docu tasks
@jan-petr Do you need to update the documentation based on these changes and the changes in #900?
JAN: Yes, definitively, please list the tasks and I will do them before merging.
HENK: OK. Task = add #900 changes and #1858 changes to the documentation :)

- [x] Verify the paths to atlases in the manual
- [x] Remove TotalWM 
- [x] In the example cases, talk about Total + WM as a replacement for the total WM. This shouldn't be a problem since we have the basic atlas as DeepWM.

### How to test

Optional: insert description about how to test the code changes here

### Release notes

Nothing. Report with #900, #1740

### PR DESCRIPTION
**Pull request by HenkMutsaerts**
### Linked issue

Closes #1858

### CODE REVIEW

File: External/Atlases4ROIs/LicensePermissive/SPM_CAT_GPL/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The *masks* in this folder all originate from SPM and/or CAT12, and thus distributed under GNU GPL.
+These are used as atlases/ROIs.
+To differentiate from /External/SPMmodified/MapsAdded, which have the same license and origin but are *maps* used for image processing in ExploreASL.
+
+DeepWM.nii: ICBM_152_nonlinear_symm pWM eroded to deep WM & resampled to 1.5 mm MNI
+Supratentorial_GM_WM.nii: same as ICBM_152_nonlinear_symm brainmask, resampled to 1.5 mm MNI (/External/SPM_modified/MapsAdded/brainmask.nii), but without structures outside the cerebrum. For which we excluded the following Mindboggle regions: 1 cerebellum exterior, 2 cerebellum WM, 10 ventral dentate nucleus, 11 basal forebrain, 12-14 cerebellar vermis. (see /CustomScripts/Atlases/Supratentorial.m; private ExploreASL repository)
+TotalGM.nii, TotalWM.nii, WholeBrain.nii: ICBM wholebrain mask, resampled to 1.5 mm MNI
```
Comment (jan-petr) [inline External/Atlases4ROIs/LicensePermissive/SPM_CAT_GPL/README.md]: We should remove TotalWM from here

File: WorkInProgress/Administration/xASL_adm_PreparePediatricTPM.m [hunk 2]
```diff
@@ -186,7 +186,7 @@
 	end
 
 	% Copy all the adult masks and transform to pediatric size
-	listMask = {'brainmask' 'TotalGM' 'TotalWM' 'WholeBrain'  'brainmask_supratentorial' 'ParenchymNarrow' 'LeftRight' 'DeepWM' 'CentralWM_QC' 'GhostSignalRatio' 'MNI_Structural'};
+	listMask = {'brainmask' 'WholeBrain'  'Supratentorial_GM_WM' 'ParenchymNarrow' 'LeftRight' 'DeepWM' 'CentralWM_QC' 'MNI_Structural'};
```
Comment (jan-petr) [inline WorkInProgress/Administration/xASL_adm_PreparePediatricTPM.m:189]: Remove totalWM from line 222.

File: External/Atlases4ROIs/LicensePermissive/SPM_CAT_GPL/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+The *masks* in this folder all originate from SPM and/or CAT12, and thus distributed under GNU GPL.
+These are used as atlases/ROIs.
+To differentiate from /External/SPMmodified/MapsAdded, which have the same license and origin but are *maps* used for image processing in ExploreASL.
+
+DeepWM.nii: ICBM_152_nonlinear_symm pWM eroded to deep WM & resampled to 1.5 mm MNI
+Supratentorial_GM_WM.nii: same as ICBM_152_nonlinear_symm brainmask, resampled to 1.5 mm MNI (/External/SPM_modified/MapsAdded/brainmask.nii), but without structures outside the cerebrum. For which we excluded the following Mindboggle regions: 1 cerebellum exterior, 2 cerebellum WM, 10 ventral dentate nucleus, 11 basal forebrain, 12-14 cerebellar vermis. (see /CustomScripts/Atlases/Supratentorial.m; private ExploreASL repository)
```
Comment (jan-petr) [inline External/Atlases4ROIs/LicensePermissive/SPM_CAT_GPL/README.md:6]: WorkInProgress/InputOutput/xASL_io_ConvertNiiGz2Mat.m: Remove TotalWM from line 11.
Sorry for putting this here, I want a nice resolvable comment ;)

**Review by jan-petr — CHANGES_REQUESTED** [review]
See also comments in the issue

File: External/Atlases4ROIs/LicensePermissive/SPM_CAT_GPL/README.md (hunk 3 above)
Comment (HenkMutsaerts) [reply External/Atlases4ROIs/LicensePermissive/SPM_CAT_GPL/README.md:6]: These WIPs are in any case wrong and not really used. But I fixed it anyway in 0741136d0b23ba106e320d13c8aeae8e92dd4910

File: WorkInProgress/Administration/xASL_adm_PreparePediatricTPM.m (hunk 2 above)
Comment (HenkMutsaerts) [reply WorkInProgress/Administration/xASL_adm_PreparePediatricTPM.m:189]: Fixed in 0741136d0b23ba106e320d13c8aeae8e92dd4910

File: External/Atlases4ROIs/LicensePermissive/SPM_CAT_GPL/README.md (hunk 1 above)
Comment (HenkMutsaerts) [reply External/Atlases4ROIs/LicensePermissive/SPM_CAT_GPL/README.md]: Fixed in 0741136d0b23ba106e320d13c8aeae8e92dd4910

**Review by jan-petr — APPROVED** [review]
OK

**Pull request merged**